manim/mobject/number_line.py

```diff
@@ -300,6 +300,14 @@ def add_numbers(self, x_values=None, excluding=None, font_size=24, **kwargs):
         self.numbers = numbers
         return numbers
 
+    def decimal_places_from_step(self, x_range):
+        if x_range is None or len(x_range) < 3:
+            return 0
```

Comment:
After moving the call to this function after x_step is defined, this can be removed since there will always be a defined x_step of 1.0 (should be changed to 1).

Reply:
done